### ISSUE TITLE
Accessibility - Improve embed menu interactions

### ISSUE THREAD

**Issue by slafleche** [vanilla/vanilla#9966]
The menu appears when you have the element selected in the rich editor, but if you tab through it, it's disorienting, because you get taken out of the editor... 

I'd like to do a little research, maybe treat it like a real menu and have better shortcuts to get back, like escape to go back to editing.

Needs some thought and research.

**Comment by slafleche** [comment]
I think what we have now is pretty good. I think we're just missing some labels and keyboard shortcuts for the menu to be a "real" menu according to the accessibility standards. See: https://www.w3.org/WAI/tutorials/menus/application-menus-code/
